airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -1261,14 +1265,36 @@ def add_to_outputs(
                 else:
                     hooks.append(f"drop view {stg_schema}.{stg_table}")
 
-                # Explicit function so that we can have type hints to satisfy the linter
-                def wrap_in_quotes(s: str) -> str:
-                    return '"' + s + '"'
-
                 config["post_hook"] = "[" + ",".join(map(wrap_in_quotes, hooks)) + "]"
             else:
                 # incremental is handled in the SCD SQL already
                 sql = self.add_incremental_clause(sql)
+        elif self.destination_sync_mode == DestinationSyncMode.overwrite:
+            if suffix == "" and not is_intermediate:
+                # drop SCD table after creating the destination table
```

Comment:
Question: This I guess will drop these scd tables at the end of any overwrite sync (reset or no reset) so we don't have to worry about them when there is a reset?
Is this correct? What happens during an incremental sync?

Reply:
This is awesome!